oops/backplane/ansa.py

```diff
@@ -284,3 +284,100 @@ def ansa_vertical_resolution(self, event_key):
 Backplane._define_backplane_names(globals().copy())
 
 ################################################################################
+
+
+
+
+################################################################################
+# UNIT TESTS
+################################################################################
+
+import unittest
+from oops.meshgrid                       import Meshgrid
+from oops.unittester_support             import TESTDATA_PARENT_DIRECTORY
+from oops.constants                      import DPR
+from oops.backplane.unittester_support   import show_info
+
+
+#===========================================================================
+def exercise_resolution(bp, printing, saving, dir, refdir,
+                        planet=None, moon=None, ring=None,
```

Comment:
That's because there are no optional arguments on the first line to line up with.  My preferred indentation actually would have been line up planet and printing, as bp has particular 
importance.